Content.Server/Doors/Components/AirlockComponent.cs

```diff
@@ -131,7 +131,7 @@ public bool IsBolted()
 
         public bool IsPowered()
         {
-            return _entityManager.TryGetComponent<ApcPowerReceiverComponent>(Owner, out var receiverComponent) && receiverComponent.Powered;
```

Comment:
pre com-dep removal it defaulted to being "powered" if there was no receiver component. I don't know if it actually matters but I assume the change was accidental so I've moved it back.